tree/ntuple/v7/src/RField.cxx

```diff
@@ -523,6 +527,9 @@ void ROOT::Experimental::Detail::RFieldBase::AutoAdjustColumnTypes(const RNTuple
       }
       SetColumnRepresentative(rep);
    }
+
+   if (fTypeAlias == "Double32_t")
+      SetColumnRepresentative({EColumnType::kSplitReal32});
```

Comment:
Should we use `EColumnType::kReal32` instead if compression is not enabled?  Also, because this happens based only on the alias type name, what if the user does, e.g.
```c++
auto f = RFieldBase::Create("f", "std::uint32_t", "Double32_t");
```
?

This is probably non-sense (and makes the `SetColumnRepresentative()` above fail), but probably we should document the behavior for such type name aliases (?).

Reply:
You're right. I guess `RFieldBase::Create` should only take one type name. It can then figure out if it is an alias or not. Taking a look.